importer/tests/test_import.py

```diff
@@ -0,0 +1,56 @@
+"""
+Tests for LORE imports.
+"""
+
+# standard library
+from unittest import TestCase
+from os.path import abspath, dirname, exists, join
+from os import mkdir
+
+# PyPi
+import requests
+
+# local
+from importer.api import import_course_from_file
+
+# TODO: Also test this course:
+# https://github.com/pmitros/edX-Insider/archive/master.zip
+def get_course_zip():
+    """
+    Get the path to the demo course.
+    """
+    url = "https://github.com/edx/edx-demo-course/archive/master.zip"
```

Comment:
We could also just use the main one edX uses: https://github.com/edx/edx-platform/tree/master/common/test/data/two_toys and copy that in here.